universal_robot/56cf07f/56cf07f.bug

```diff
@@ -73,3 +73,8 @@ fix:
   commits:
     - repo: https://github.com/ros-industrial/universal_robot
       hash: 56cf07f62fcdd3927ebff216da281d3b975adefc
+fault-codes:
+  - BDO:DEPENDENCY
+failure-codes:
+  - SOFTWARE:BUILD
+  - SYSTEM:NONE
```

Comment:
How did this pass the `yamllint` test before?

Reply:
From [here](https://travis-ci.com/github/robust-rosin/robust/builds/216629561)
```bash
./universal_robot/56cf07f/56cf07f.bug:31:81: [warning] line too long (88 > 80 characters) (line-length)
./universal_robot/56cf07f/56cf07f.bug:34:81: [warning] line too long (83 > 80 characters) (line-length)
./universal_robot/56cf07f/56cf07f.bug:36:81: [warning] line too long (85 > 80 characters) (line-length)
./universal_robot/56cf07f/56cf07f.bug:37:81: [warning] line too long (86 > 80 characters) (line-length)
./universal_robot/56cf07f/56cf07f.bug:38:81: [warning] line too long (116 > 80 characters) (line-length)
```
No changes were done to these lines here